migrations/V217__Add MTO_Common_Solution_Tag_Type.sql

```diff
@@ -0,0 +1,22 @@
+-- Add new tag type for MTO common solutions
+ALTER TYPE tag_type ADD VALUE 'MTO_COMMON_SOLUTION';
+
+/* 
+  Add the new UUID/id column. This is used to allow MTO common solutions to use the entity_uuid on tagged content, same as user_account.
+  Previously operational solutions were using int as id, moving to UUID allows for better compatibility with other tables 
+*/
+ALTER TABLE mto_common_solution
+ADD COLUMN id UUID;
+
```

Comment:
thanks Patrick, would you mind changing this to a sql column comment? That way it will persist beyond a flyway migration
```suggestion
COMMENT ON COLUMN mto_common_solution.id IS 'This is used to allow MTO common solutions to use the entity_uuid on tagged content, same as user_account.  Previously operational solutions were using int as id, moving to UUID allows for better compatibility with other tables ';
```